site/src/util/mdxComponents.tsx

```diff
@@ -99,6 +108,21 @@ export const mdxComponents: Record<string, React.ReactNode> = {
       <a id={props.id} />
     );
   },
+  table: ({ children, ...props }: HTMLProps<HTMLTableElement>) => (
+    <Box
+      sx={{
+        "td, th": {
+          border: ({ palette }) => `1px solid ${palette.gray[30]}`,
+          paddingY: 1,
+          paddingX: 3,
+          typography: "bpSmallCopy",
+        },
+        marginBottom: 2,
+      }}
+    >
+      <table {...props}>{children}</table>
```

Comment:
We don't have to start using this now, but there is a [MUI Table component](https://mui.com/components/tables/) that'll probably give us most of the styling we need for free.